RELEASES.md

```diff
@@ -4,6 +4,9 @@
 
 ## 3.1.0 (Feb 14th, 2024)
 
+### Web snippet additional update to 1.1.1
```

Comment:
This should be above the ## 3.1.0 as it is it's own release and not part of the 3.1.0 release

Reply:
Everything else looks good